extensions/binding/org.eclipse.smarthome.binding.fsinternetradio/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,86 @@
+            <channel id="play-info-text" typeId="play-info-text"/>
+            <channel id="preset" typeId="preset"/>
+        </channels>
+
+        <config-description>
+            <parameter name="ip" type="text" required="true">
+                <label>Internet Radio Address</label>
+                <description>The IP address (name or numeric) of the internet radio.</description>
+            </parameter>
+            <parameter name="port" type="integer" required="true">
+                <label>Internet Radio Port</label>
+                <description>The port of the internet radio (default: 80).</description>
+                <default>80</default>
+            </parameter>
+            <parameter name="pin" type="text" required="true">
+                <label>Internet Radio Pin</label>
+                <description>The PIN configured in the internet radio (default: 1234).</description>
+                <default>1234</default>
+            </parameter>
+            <parameter name="refresh" type="integer">
+                <label>Refresh interval</label>
+                <description>Specifies the refresh interval in seconds.</description>
+                <default>60</default>
+            </parameter>
+        </config-description>
+    </thing-type>
+
+    <channel-type id="power">
+        <item-type>Switch</item-type>
+        <label>Internet Radio Power</label>
+        <description>Switch the radio on or off.</description>
+        <category>Switch</category>
+    </channel-type>
+    <channel-type id="preset">
+        <item-type>Number</item-type>
+        <label>Internet Radio Preset</label>
+        <description>Preset radio stations configured in the radio.</description>
+    </channel-type>
+    <channel-type id="volume">
+        <item-type>Number</item-type>
+        <label>Internet Radio Volume</label>
+        <description>Radio volume (min=0, max=32).</description>
+        <category>SoundVolume</category>
+        <state min="0" max="32" step="1" />
+    </channel-type>
+    <channel-type id="mute">
+        <item-type>Switch</item-type>
+        <label>Internet Radio Mute</label>
+        <description>Mute the radio.</description>
+    </channel-type>
+    <channel-type id="play-info-name">
+        <item-type>String</item-type>
+        <label>Internet Radio play info name</label>
+        <description>The name of the current radio station or track.</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="play-info-text">
+        <item-type>String</item-type>
+        <label>Internet Radio play info text</label>
+        <description>Additional information e.g. of the current radio station.</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="mode">
+        <item-type>Number</item-type>
+        <label>Internet Radio Mode</label>
+        <description>The radio mode, e.g. FM radio, internet radio, AUX, etc.</description>
```

Comment:
Is there any standardized option list of these modes? Or a min or max? Step should be probably set to 1 at least.